README.md

```diff
@@ -97,6 +97,25 @@ npm run build
 
 Generated files of the website go to the `.next` folder.
 
+### Running the Docker image
+
+To run the Docker image, simply run the following command:
+
+### Steps to start server
```

Comment:
better name the image like `asyncapi-website` in instructions, so it is clear what it is when someone builds it, run, and see on a list of local images